SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/DxeTcg2PhysicalPresenceLib.c

```diff
@@ -680,35 +680,56 @@ Tcg2HaveValidTpmRequest  (
       break;
 
     case TCG2_PHYSICAL_PRESENCE_SET_PCR_BANKS:
-      if ((Flags.PPFlags & TCG2_BIOS_TPM_MANAGEMENT_FLAG_PP_REQUIRED_FOR_CHANGE_PCRS) == 0) {
-        *RequestConfirmed = TRUE;
+      // MU_CHANGE_108842
+      // MU_CHANGE [BEGIN] - Do not allow Flags to bypass confirmation in production mode.
+
+      if (PcdGetBool (PcdEnableShipMode)) {

```

Comment:
Can we name this PCD something relevant to what it enables instead of what is enabling it?